activerecord-bitemporal.gemspec

```diff
@@ -14,6 +14,7 @@ Gem::Specification.new do |spec|
   spec.description   = %q{Enable ActiveRecord models to be handled as BiTemporal Data Model.}
   spec.homepage      = "https://github.com/kufu/activerecord-bitemporal"
   spec.license       = "Apache 2.0"
+  spec.required_ruby_version = ">= 2.7.0"
```

Comment:
This is should be worked as another PR.

Reply:
I opened another PR for adding this change.
https://github.com/kufu/activerecord-bitemporal/pull/125